Oracle/README.md

```diff
@@ -5,15 +5,63 @@ so it can be migrated to [Snowflake](https://www.snowflake.com/) using [SnowConv
+
+```bash
+sqlplus -v
+```
+
+If SQL*Plus is installed and properly configured, you will see the version information displayed on the screen. This confirms that SQL*Plus is installed and ready to be used for executing SQL commands and scripts.
+
+* You will need the conection string to your database. If your database is hosted on AWS, you will need to use the connection format provided in this link: [AWS conection string](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_ConnectToOracleInstance.SQLPlus.html). An example of the connection string could be: 
+  `TEST_USER@(DESCRIPTION=(ADDRESS=(PROTOCOL=TCP)(HOST=url.amazonaws.com)(PORT=1521))(CONNECT_DATA=(SID=orcl)))` 
+
+The link contains detailed instructions on how to connect to an Oracle instance on AWS using SQL*Plus or SQLcl. It provides the necessary format for specifying the connection details, such as the hostname, port, and SID. By following the instructions in the link, you will be able to establish a successful connection to your Oracle database hosted on AWS.
+* It is recommended to use a user  with **sysadmin** privileges.
+
 
 ## Usage
 
 The following are the steps to execute the DDL Code Generation. They can be executed in Linux/Unix and Windows enviroments.
 
 ## **For Linux/Unix:**
 
-1 - Modify `createddls.sh` – Using a text editor modify the following parameters:
+There are two options available to execute the extraction script based on your environment and the tools you have installed: Sql*Plus and sqlcl.
+
+Sql*Plus: If you have Oracle installed on your machine, you can use Sql*Plus, which is a command-line tool for Oracle Database. It provides a powerful and interactive interface to execute SQL statements and scripts. 
+
+sqlcl: If you don't have Oracle installed locally or prefer a modern command-line interface, you can use sqlcl (SQL Developer Command Line). Sqlcl is a free, lightweight tool provided by Oracle that offers a feature-rich SQL and PL/SQL scripting environment. It provides similar functionality to Sql*Plus but with additional enhancements. 
+
+### Using SQLcl
+
+1. Download the SQLcl software from: https://www.oracle.com/database/sqldeveloper/technologies/sqlcl/download/
+2. Extract the files in any directory that you choose.
+3. Modify `create_ddls.sh` – Using a text editor modify the following parameters, for more details about the parameters see the [Notes area](#notes):
+
+* `ORACLE_SID`
+* `CONNECT_STRING`
+* `SCRIPT_PATH`
+* `SQLCL_PATH`
+  
+4. After modifying, the `create_ddls.sh` file can be run from the command line to execute the extract. 
+   In order to run the script you can open a terminal in the  `create_ddls.sh` path  and execute the command:
+   ```bash
+   source create_ddls.sh
+   ```
+   or
+   ``` bash
+   sh create_ddls_sqlcl.sh
+   ``` 
+   The following files will be created in the directory `/<SCRIPT_PATH>/object_extracts/DDL`.

```

Comment:
Are these all the files that will be created? DDL is the name of the folder, right? but I think the files generated are something like TAbles, Views...